kolibri/plugins/html5_app_renderer/assets/src/vue/index.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+
+  <div v-el:container class="container" allowfullscreen>
+    <icon-button
+      class="btn"
+      :text="inFullscreen ? $tr('exitFullscreen') : $tr('enterFullscreen')"
+      @click="togglefullscreen">
+    </icon-button>
+    <iframe v-el:sandbox class="sandbox" :src="rooturl" sandbox="allow-scripts"></iframe>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+
+    components: {
+      'icon-button': require('kolibri.coreVue.components.iconButton'),
+    },
+
+    props: ['defaultFile'],
```

Comment:
it would be best to describe the props with additional checks. E.g.:

```javascript
props: {
  defaultFile: {
    type: String,
    required: true,
  }
}
```